### PR TITLE
LPS-156890 | Can get ManyToMany relationship details as nested fields_5

### PR DESCRIPTION
**Pull request by Hijky**
**Background**
Add a test to assert get ManyToMany relationship details as nested fields_5

**Test Plan**
Given university created
And Given subject created
And Given manyToMany relationship universitySubjects created
And Given two university entries created
And Given two subject entries created
When each subject related to each universtity
Then I receive correct information about the related objects o/c/subjects?nestedFields=universitiesSubjects

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-156890

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:headless-frontend-functional:nocompile

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro [hunk 1]
```diff
@@ -410,7 +410,7 @@ definition {
 	}
 
 	macro assertResponseIncludeCorrectDetailsOfNotDeletedObject {
-		var actual = JSONUtil.getWithJSONPath("${responseToParse}", "$..universitiesSubjects[?(@.id==${subjectId})].name");
+		var actual = JSONUtil.getWithJSONPath("${responseToParse}", "$..universitiesSubjects[?(@.id==${objectEntryId})].name");
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro:413]: I like the refactor! 👏🏻

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/AdvancedObjectAPI/SupportManyToManyRelationships.testcase [hunk 2]
```diff
@@ -266,6 +266,72 @@ definition {
...
+				objectEntry2 = "${subjectId2}",
+				relationshipName = "universitiesSubjects");
+		}
+
+		task ("Then receiving correct information about the related objects in subjects GET endpoint with nestedFields=universitiesSubjects") {
+			var responseToParse = ObjectDefinitionAPI.getObjectsWithNestedField(
+				nestedField = "universitiesSubjects",
+				objects = "subjects");
+
+			ObjectDefinitionAPI.assertResponseIncludeCorrectDetailsOfNotDeletedObject(
```
Comment (magjed4289) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/AdvancedObjectAPI/SupportManyToManyRelationships.testcase:323]: I miss checking that each subject is related to its respective university. Does the response give this information?

**Review by magjed4289 — CHANGES_REQUESTED** [review]

**Review by magjed4289 — APPROVED** [review]

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:headless-frontend-functional:nocompile

**Pull request closed without merging**